### DISCUSSION TITLE
Recommend approach to obtain 3D scene object state while dragging

### DISCUSSION THREAD

**Discussion by h0uter** [zauberzeug/nicegui#3450] [Ideas / Feature Requests]
While using the draggable elements in the 3D scene we can define on `drag_start` and `drag_end` callbacks. However it would be useful if we could obtain the state of the 3D scene element while we are dragging it so we can update other 3D scene elements in tandem. 

## Example
The following code snippet illustrates that the state of the dragged element is only updated on release.

```py
from nicegui import ui

lbl = ui.label("sphere @")


def update_label():
    lbl.set_text(f"sphere @ {sphere.x}, {sphere.y}, {sphere.z}")


ui.timer(0.01, update_label)


with ui.scene() as scene:
    sphere = scene.sphere().move(z=1).draggable()


ui.run()
```

https://github.com/user-attachments/assets/a641f687-6de0-475c-b3fb-3a40c7cfa00d

## Why would this be usefull
An example usecase for being able to read the state mid-drag is that it would allow end users to spatially mark areas by dragging the corner of a polygon while instantly updating that polygon to match the users selection. For example to take slices of 3D objects or to define a geofence.

**Comment by frankvp11** [comment]
I believe that if you remove the line found https://github.com/zauberzeug/nicegui/blob/277ada690e2b61d8528e8b6037d410c643ce196c/nicegui/elements/scene.js#L152 which returns if the event.type is drag, it would most likely do what you require, which is emit drag events much more frequently.
The problem with this might be that it would cause lag, so be weary of that. Note that I haven't actually tested it, but it seems like it'd work

**Comment by falkoschindler** [comment]
Thanks for pointing out this limitation, @h0uter!
I just created PR #3466 to allow subscribing to the "drag" event.

**Comment by falkoschindler** [comment]
With PR #3466 being released in version 1.4.35, I'll close this feature request as completed.
